plugin/storage/kafka/options.go

```diff
@@ -64,6 +65,10 @@ func (opt *Options) AddFlags(flagSet *flag.FlagSet) {
 		configPrefix+suffixTopic,
 		defaultTopic,
 		"(experimental) The name of the kafka topic")
+	flagSet.String(
+		configPrefix+suffixProtocolVersion,
+		"",
+		"(experimental) Kafka protocol version - must be supported by kafka server")
```

Comment:
why experimental?

Reply:
just sticking to convention. all other kafka.producer settings are marked as experimental

```
      --kafka.producer.brokers string                 (experimental) The comma-separated list of kafka brokers. i.e. '127.0.0.1:9092,0.0.0:1234' (default "127.0.0.1:9092")
      --kafka.producer.encoding string                (experimental) Encoding of spans ("json" or "protobuf") sent to kafka. (default "protobuf")
      --kafka.producer.protocol-version string        (experimental) Kafka protocol version - must be supported by kafka server
      --kafka.producer.topic string                   (experimental) The name of the kafka topic (default "jaeger-spans")
```

Reply:
oh, that's probably just stale, can you remove (experimental) from all 4?